CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [2.0]
+### Changed
+- **Breaking change:** Replaced "positive/nagative button" clicked listener by global listener which need to be implemented by host activity
```

Comment:
Maybe

> Replaced "positive/nagative button" clicked listener by global listener which need to be implemented by host activity

->

> Replaced "positive/negative button" clicked listener by a global listener which needs to be implemented by the host activity

?

Reply:
right, thank for correcting my English :)